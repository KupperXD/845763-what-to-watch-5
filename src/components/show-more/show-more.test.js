import React from 'react';
import renderer from 'react-test-renderer';
import {BrowserRouter} from "react-router-dom";

import ShowMore from "./show-more";

const noop = () => {};

it(`renders correctly`, () => {
  const showMore = renderer
    .create(
        <BrowserRouter>
          <ShowMore
            isShow={true}
            clickHandler={noop}
          />
        </BrowserRouter>
    )
    .toJSON();

  expect(showMore).toMatchSnapshot();
});
