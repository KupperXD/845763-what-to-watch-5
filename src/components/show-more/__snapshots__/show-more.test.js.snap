// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="catalog__more"
  style={
    Object {
      "display": "block",
    }
  }
>
  <button
    className="catalog__button"
    type="button"
  >
    Show more
  </button>
</div>
`;
